ext/crypto/00_crypto.js

```diff
@@ -691,7 +691,8 @@
             algorithm: "AES-GCM",
             length: key[_algorithm].length,
             iv: normalizedAlgorithm.iv,
-            additionalData: normalizedAlgorithm.additionalData,
+            additionalData: normalizedAlgorithm.additionalData ||
```

Comment:
Maybe use `??` instead of `||`. Same for L3829

Not sure what the Deno standard on this is, but `??` shows intent as default value.

Reply:
Doesn't make any difference. webidl takes care of the types so `normalizedAlgorithm.additionalData` can only be `undefined` or a uint8array. I don't think Deno has any standard on this. I would've preferred to just pass `null` to serde here because defaults are set Rust side.